contracts/ProofOfExistence2.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.4.4;
+
+contract ProofOfExistence2 {
+    bytes32[] private proofs;
+
+    function storeProof(bytes32 proof) {
+        proofs.push(proof);
```

Comment:
How does this work? I tried using the `push` function but it is apparently reserved for `storage` memory types? I tried pushing to `address[] addressList` and i get compile errors.